src/main/scala/io/iohk/ethereum/db/dataSource/IodbDataSource.scala

```diff
@@ -1,23 +1,46 @@
 package io.iohk.ethereum.db.dataSource
 
+import java.io.File
 import java.nio.ByteBuffer
 import java.util.concurrent.atomic.AtomicLong
 
 import io.iohk.iodb.{ByteArrayWrapper, LSMStore}
 
-class IodbDataSource(lSMStore: LSMStore) extends DataSource {
+class IodbDataSource private (lSMStore: LSMStore,
+                              keySize: Int,
+                              path: String) extends DataSource {
```

Comment:
I'm not requiring this change, but it is recommended to keep multiline parameters on their own lines. Main reason is that when you change the name you don't have re-indent. http://docs.scala-lang.org/style/declarations.html#classes